src/extension.ts

```diff
@@ -41,6 +42,7 @@ export function activate(context: vs.ExtensionContext) {
 		return; // Don't set anything else up; we can't work like this!
 	}
 	context.globalState.update(stateLastKnownSdkPathName, dartSdkRoot);
+	debug.writeSdkPath(dartSdkRoot); // Write the SDK path for the debugger to find.
```

Comment:
Yeah, this needs doing for more than just this though - I created #18 to handle this. We'll need to terminate and restart the analyzer, etc.

I also added a note in that case about this, so I think we can just deal with it there (I think it should be fixed by v1).